src/Symfony/Component/Config/Definition/Builder/BuilderAwareInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Config\Definition\Builder;
+
+/**
+ * An interface that can be implemented by nodes which build other nodes.
+ *
+ * @author Roland Franssen <franssen.roland@gmail.com>
+ */
+interface BuilderAwareInterface
+{
+    /**
+     * Sets a custom children builder.
```

Comment:
is this really needed :p ?

Reply:
It can be just comment, and not phpdoc IMHO.

Reply:
I only moved things around, https://github.com/symfony/symfony/pull/26335/files#diff-b579be54110e0018ae82a55725bc373fL49

works for me as is :)

Reply:
We use phpdoc for documenting methods, not wild comments. So it's either a docblock or nothing at all, personally I would remove it as it does not bring much IMHO.